pubspec.lock

```diff
@@ -588,13 +581,6 @@ packages:
       url: "https://pub.dartlang.org"
     source: hosted
     version: "0.1.0"
-  xml:
-    dependency: transitive
-    description:
-      name: xml
-      url: "https://pub.dartlang.org"
-    source: hosted
-    version: "3.6.1"
 sdks:
-  dart: ">=2.7.0 <3.0.0"
+  dart: ">=2.9.0-14.0.dev <3.0.0"
   flutter: ">=1.12.13+hotfix.6 <2.0.0"
```

Comment:
Are the changes to this pubspec.lock file autogenerated? Should this be included in the PR or deleted so it doesn't interfere with other people's versions/systems?

Reply:
Yup - autogenerated!